runtime/etc/src/mli/client_runtime.c

```diff
@@ -81,30 +95,50 @@ void chpl_library_init(int argc, char** argv) {
 
   // Just hardcode these values for now.
   const char* ip = "localhost";
-  const char* setupPort = "5554";
-  const char* mainport = "5555";
-  const char* argport = "5556";
-  const char* resport = "5557";
 
   if (initialized) { return; }
   initialized = 1;
 
   // Set up the clientside ZMQ sockets.
   chpl_mli_client_init(&chpl_client);
 
-  chpl_mli_bind(chpl_client.setup_sock, ip, setupPort);
+  chpl_mli_bind(chpl_client.setup_sock, ip, "*");
```

Comment:
I think I wrote some logic that converts "localhost" -> "*" in `chpl_mli_bind`, not sure that's needed now if we're just going to pass an ephemeral device in directly.

Reply:
Yeah, I debated about updating bind as well as connect.  I think it would be reasonable to just pass in *:*, or even just call bind with only the socket as an argument, since we won't be using explicit port and hostname information for that call at all

Reply:
Yeah, I suppose some refactoring is in order for those functions.